node/src/types/block.rs

```diff
@@ -1111,6 +1113,226 @@ impl Item for BlockHeaderWithMetadata {
         self.block_header.height()
     }
 }
+#[derive(DataSize, Debug, Clone, Copy, Serialize, Deserialize, PartialEq, Eq, Hash)]
+/// ID identifying a request for a batch of block headers.
+pub struct BlockHeadersBatchId {
```

Comment:
I think all the new `pub` structs in this file and their fields and methods can be `pub(crate)` instead.

Reply:
Done in [`36ded88` (#3020)](https://github.com/casper-network/casper-node/pull/3020/commits/36ded88067816ec81e9ed1af5e997cf372fb127c)